src/cdk-lib/bedrock/inference-profiles/cross-region-inference-profile.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+import { Arn, ArnFormat, Aws } from 'aws-cdk-lib';
+import { Grant, IGrantable } from 'aws-cdk-lib/aws-iam';
+import { BedrockFoundationModel, IInvokable } from '../models';
+import { IInferenceProfile, InferenceProfileType } from './common';
+
+export enum CrossRegionInferenceProfileRegion {
+  /**
+   * Cross-region Inference Identifier for the European area.
+   * According to the model chosen, this might include:
+   * - Frankfurt (`eu-central-1`)
+   * - Ireland (`eu-west-1`)
+   * - Paris (`eu-west-3`)
+   */
+  EU = 'eu',
+  /**
+   * Cross-region Inference Identifier for the United States area.
+   * According to the model chosen, this might include:
+   * - N. Virginia (`us-east-1`)
+   * - Oregon (`us-west-2`)
+   * - Ohio (`us-east-2`)
+   */
+  US = 'us',
+  /**
+   * Cross-region Inference Identifier for the Asia-Pacific area.
+   * According to the model chosen, this might include:
+   * - Tokyo (`ap-northeast-1`)
+   * - Seoul (`ap-northeast-2`)
+   * - Mumbai (`ap-south-1`)
+   * - Singapore (`ap-southeast-1`)
+   * - Sydney (`ap-southeast-2`)
+   */
+  APAC = 'apac',
```

Comment:
How about just `AP`?

```suggestion
  AP = 'ap',
```

Reply:
Unfortunately the cross-inference regions do not match with the region prefix, in inference profiles this is called APAC 
![Screenshot 2024-11-20 at 18 49 43](https://github.com/user-attachments/assets/a3ded537-f896-4940-b8cc-70f61eb858d3)